boards/stm32f746g-disco/board.c

```diff
@@ -18,9 +18,18 @@
  * @}
  */
 
+#include "kernel_defines.h"
 #include "board.h"
 #include "periph/gpio.h"
 
 void board_init(void)
 {
+    if (IS_USED(MODULE_PERIPH_LTDC)) {
+        gpio_init(LCD_DISP_PIN, GPIO_OUT);
+        gpio_set(LCD_DISP_PIN);
+    }
+    else {
+        gpio_init(BACKLIGHT_PIN, GPIO_OUT);
+        gpio_clear(BACKLIGHT_PIN);
```

Comment:
Is the backlight controlled by  LTDC or why do we only need it if the display is *not* used?

Reply:
That's a small change that indeed turns off the screen if it's unused, saving some power. The backlight is not controller by the LTDC, it's an external pin that is controlled by user code (like here).

Reply:
You are right the logic is odd here. Seems like the backlight is enabled by default. `BACKLIGHT_ON/OFF` macros are broken if it's not always initialized. Should be better now.